### ISSUE TITLE
Add iOS TimeTool View Implementation

### ISSUE THREAD

**Issue by maximbircu** [maximbircu/devtools-library#24]
## Abstract
Sometimes, we might have time as a configuration value. Sure thing that a timestamp most probably will be of a Long time. Thus, a TextTool could be used to represent it. Though, using the text tool, in this case, might not be handy, especially for high values. 

A TimeTool was implemented in #20. It helps the library consumer manipulate easily time tool values by adding a good UX in the form of a time picker, on Android. We should ad a TimeTool view implementation for iOS too.

## Requirements
Add a view implementation for TimeTool.

<img src="https://user-images.githubusercontent.com/12527390/78816200-11ff5900-79da-11ea-84b7-245952800162.png" width=100>

## Implementation details
The time tool logic is already implemented inside the common module, you just need to provide a view implementation for it.
So, basically you need to:
- Provide an implementation for [TimeToolView](https://github.com/maximbircu/devtools-library/blob/master/devtools/common/src/commonMain/kotlin/com/maximbircu/devtools/common/presentation/tools/time/TimeToolView.kt)
- Consume the [TImeTooPresenter](https://github.com/maximbircu/devtools-library/blob/master/devtools/common/src/commonMain/kotlin/com/maximbircu/devtools/common/presentation/tools/time/TimeToolPresenter.kt)

## Consumer flow
1. Open the sample app and go to the YML dev tools screen.
2. Find the TimeTool UI control 
3. Try to update the configuration value and make sure it's persisted after app restart.

Repeat the same use-case for Memory dev tools screen.
